src/Constants.js

```diff
@@ -0,0 +1,59 @@
+const MISSION_UTILS = require("@woowacourse/mission-utils");
+const CONSOLE_UTIL = MISSION_UTILS.Console;
+const RANDOM_UTIL = MISSION_UTILS.Random;
+
+const FROM_ONE_TO_FORTYFIVE = Array.from(
+  { length: 45 },
+  (VOID, index) => index + 1
+);
```

Comment:
만약 `VOID`를 사용하지 않아서 그렇게 작성하신거라면 아래처럼 `_`로 대체할 수 있어요!
```suggestion
const FROM_ONE_TO_FORTYFIVE = Array.from(
  { length: 45 },
  (_, index) => index + 1
);
```